plugins/bulk-import/src/components/AddRepositories/AddRepositoriesFormFooter.tsx

```diff
@@ -38,33 +43,39 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+const sPad = (repositories: AddedRepositories) =>
+  (Object.keys(repositories || [])?.length || 0) > 1 ? 's' : '';
+
 export const AddRepositoriesFormFooter = () => {
   const styles = useStyles();
-  const { values, handleSubmit } =
+  const { values, handleSubmit, isSubmitting } =
     useFormikContext<AddRepositoriesFormValues>();
   const submitTitle =
     (values.approvalTool === ApprovalTool.Git
       ? 'Create pull request'
-      : 'Create ServiceNow ticket') +
-    (((values.repositories && Object.keys(values.repositories)?.length) || 0) >
-    1
-      ? 's'
-      : '');
+      : 'Create ServiceNow ticket') + sPad(values.repositories);
+
+  const disableCreate =
+    !values.repositories || Object.values(values.repositories).length === 0;
+
+  const toolTipTitle = () => {
+    if (disableCreate) {
+      return 'Please wait until the catalog-info.yaml files are generated';
+    }
+    return '';
```

Comment:
I'm sorry, but your new code will not work because you're checking toolTipTitle, which is an always-defined function.

Please check for `disableCreate` in the return statement. Remove the `toolTipTitle` function and set the title directly to the string that this function returns.

Reply:
Updated code should work